### PR TITLE
Added Manipulator Trigger

### PR DESCRIPTION
**Pull request by Dar1ngit**
This Pull Request closes issue #

Description of the changes:
 - 

 - [ ] I have labelled this issue with the priority and correct subsystem.
 - [ ] I set the person that I worked on this with as a reviewer (who will need to approve the changes)

### CODE REVIEW

**Review by SBsteveyHops — APPROVED** [review]
I do not see any problems with the code but it looks like the "Format" check failed again.

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -39,6 +43,8 @@ public RobotContainer() {
 
     // Configure the trigger bindings
     configureBindings();
+
+    new JoystickButton(controllerHID, 9).onTrue(new GrabManipulatorCommand(manipulator));
```
Comment (LinusSkucas) [inline src/main/java/frc/robot/RobotContainer.java:47]: Could you stick this in the `configureBindings` method? Just to keep things clean.

**Review by LinusSkucas — CHANGES_REQUESTED** [review]
Just 1 tiny thing

**Comment by LinusSkucas** [comment]
Closes #7

**Review by AG-Coding — CHANGES_REQUESTED** [review]

**Review by SBsteveyHops — APPROVED** [review]
I don't see any problems with the changes.

**Review by LinusSkucas — APPROVED** [review]

**Review by AG-Coding — APPROVED** [review]

**Pull request merged**